src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -27,9 +27,13 @@ import {getShortName, localizeCountry} from '../location_formatting';
 
 const FLAG_IMAGE_DIR = 'images/flags';
 
+export interface RegionPickerOption extends CloudLocationOption {
```

Comment:
Please add a TODO to remove the dependency on the model.
Or define this type independently. The caller should convert from the domain model to the view model.

Reply:
I added a TODO.

Even if we remove the dependency on `CloudLocationOption`, I think we still want to share the `CloudLocation` type across the whole codebase, so I don't think it's as simple as "remove the dependency".  However, I also think this code may change: for GCP, we really should treat this as a "location picker", and let the server creation flow choose the zone _after_ the location (i.e. GCP Region) is selected.  That would help with a problem I've encountered several times, where the selected zone is "available" but actually has no capacity when we try to create the instance.